app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,5 @@
+
+<% @ingredients.sorted.each do |ing| %>
```

Comment:
Since you got to this extension, I'd probably override the original `@ingredient` in the controller with `Ingredient.sorted`. This works, but you could get rid of the original too once the User Story changed the requirements.